ActindoWorkFlow/Overview/02_WorkflowProcessElements.md

```diff
@@ -0,0 +1,199 @@
+[!!Workflows](ActindoWorkFlow)
+
+# Workflow and process elements
+
+![Workflow](/Assets/Screenshots/ActindoWorkFlow/Workflows/Workflow.png "[Workflow]")
+
+The *Workflows* module allows customers to model their own processes to match their companies structure. Such a workflow consists of a number of fixed elements that are chained together, each of them having specific characteristics. These elements are used to design the workflow, which can be edited as necessary according to customer needs. For detailed information, see [Manage workflows](/ActindoWorkFlow/Operation/01_ManageWorkflows.md).
+
+A workflow runs from a fixed start point to a fixed end point, which can be in turn linked to another workflow. A workflow must contain at least one action between the start and the end point, which are a special type of places. The actions are linked through places with each other, connecting the corresponding action input and output ports. All these elements are described more in detail below.
+
+The *Workflows* module is linked to different system modules and plugins, such as *Invoicing*, *Email*, *OmniChannels*, *Tasks*, among others. The actions available to create workflows depend on the modules installed in the system.
+
+Since business processes, and the workflows that map them, evolve over time, workflows can be versioned. While every workflow can be uniquely identified through its key, there may be any number of versions of it. However, at a given point in time, only **one** of those versions may be published. When a process for a given workflow is to be started, that published version of the workflow is used. This allows to keep modelling a certain workflow without interrupting the daily business.
```

Comment:
```suggestion
Since business processes, and the workflows that map them, evolve over time, workflows can be versioned. While every workflow can be uniquely identified through its key, there may be any number of versions of it. However, at a given point in time, only one of those versions may be published. When a process for a given workflow is to be started, that published version of the workflow is used. This allows to keep modelling a certain workflow without interrupting the daily business.
```
keine hervorhebung von wörtern (nur ui elemente)